bin/oref0-setup.sh

```diff
@@ -686,6 +692,16 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
             sudo make install || die "Couldn't make install bluez"
             killall bluetoothd &>/dev/null #Kill current running version if its out of date and we are updating it
             sudo cp ./src/bluetoothd /usr/local/bin/ || die "Couldn't install bluez"
+            
+            # Replace all other instances of bluetoothd and bluetoothctl to make sure we are always using the self-compiled version
+            while IFS= read -r bt_location; do 
+                if [[ $($bt_location -v|awk -F': ' '{print $NF}') != "5.48" ]]; then
+                    killall bluetoothd &>/dev/null #Kill current running version if its out of date and we are updating it
+                    sudo cp -p $(find $HOME/src/bluez-5.48 -name $(basename $bt_location)) $bt_location || die "Couldn't replace $(basename $bt_location) in $(dirname $bt_location)"
+                    touch /tmp/reboot-required
+                fi
+            done < <(find / -name "bluetoothctl" -o -name "bluetoothd")
```

Comment:
This is some advanced bash.  Can you explain what it does?

Reply:
looks for all files named bluetoothd OR bluetoothctl throughtout the filesystem and then runs a `-v` on them to get the version... if they don't match 5.48, it will go and find the same named file under $HOME/src/bluez-5.48 and copy to the location. 

> Example: It finds a bluetoothd file in /home/.rootfs/usr/local/bin/ for instance that reports it is version 5.23, it will go find bluetoothd in $HOME/src/bluez-5.48 and copy it to /home/.rootfs/usr/local/bin/

Reply:
Nice.  :-)

Reply:
Would it be possible to change the != to a "less than" so that any new version of bluethooth > 5.49 is also allowed?

Reply:
looking into how to accomplish this, bash apparently doesn't like decimals

Reply:
How about just a string based sort comparison?

Reply:
I'm commiting the change, I need to run through it again to verify it works as intended, works good as a I spot checked the logic/syntax. Will update with my findings

Reply:
Works like a dream